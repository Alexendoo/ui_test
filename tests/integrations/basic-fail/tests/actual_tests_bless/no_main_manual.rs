//@compile-flags: --crate-type=bin

pub fn foo() {}
