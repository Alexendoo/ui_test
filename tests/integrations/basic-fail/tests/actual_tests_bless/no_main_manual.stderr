error: cannot mix `bin` crate type with others

error[E0601]: `main` function not found in crate `no_main_manual`
 --> tests/actual_tests_bless/no_main_manual.rs:3:16
  |
3 | pub fn foo() {}
  |                ^ consider adding a `main` function to `tests/actual_tests_bless/no_main_manual.rs`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0601`.
